### ISSUE TITLE
Textarea doesn't work in IE and Firefox

### ISSUE THREAD

**Issue by demisx** [angular/material#1156]
I have this textarea field:

```html
  <md-input-container>
    <label>Description</label>
    <textarea ng-model="post.description"></textarea>
  </md-input-container>
```

It works OK in Chrome and Safari, but doesn't work in IE 11 and Firefox 34. In IE it is always a one-liner, in FF it is a 2-line field.

**Comment by RomainGx** [comment]
Moreover, if you type many rows in the textarea on Chrome (enough for the scrollbar to appear), hitting any keyboard button will scroll the page up automatically.
Thus, writing a long text is very uncomfortable for the user.

**Comment by pcsforeducation** [comment]
+1, this is really easy to see on https://material.angularjs.org/#/demo/material.components.input if you copy/paste a large block of text into the textarea and try to type at the end.

**Comment by DeGuitard** [comment]
Found this workaround:
```md-input-container .md-input { flex: none !important; }```

Tested in Firefox 36, Chromium 43 and Safari 7.1.4.
Couldn't find a side-effect, so a solution might be to remove `flex: 1` from input.css.

**Comment by tinusn** [comment]
Had the same issue and the fix suggested by @DeGuitard seemed to do the trick.

**Comment by davidwadge** [comment]
This workaround doesn't work for IE10 - remains fixed as a single line.

**Comment by topherfangio** [comment]
It looks like this was closed by #1157 which was merged a long time ago. I am not able to reproduce it on Firefox/IE on 0.10.1 or master.

Can you guys confirm if this is already fixed or if it is still causing issues?

**Comment by DeGuitard** [comment]
I confirm, now it works properly on FF40 / IE11 / Edge 20.10240.16384.
